tests/ssg_test_suite/xml_operations.py

```diff
@@ -184,6 +185,18 @@ def add_platform_to_benchmark(root, cpe_regex):
             benchmark.insert(platform_index, e)
 
 
+def add_product_to_fips_certified(root, product="fedora"):
+    query = OVAL_DEF_QUERY + "/{0}".format(
+        "oval-def:definition[@id='oval:ssg-installed_OS_is_FIPS_certified:def:1']/"
+        "oval-def:criteria")
```

Comment:
I think you can just concatenate them without using the format method.

Reply:
I could. I just kept it consistent with https://github.com/ComplianceAsCode/content/blob/e033559db69f4b4217f2fdbbf39afbb44a094cab/tests/ssg_test_suite/xml_operations.py#L84 aka copy-paste :)